### PR TITLE
feat(mux-uploader, mux-uploader-react): Add pausable functionality and pause button component.

### PR DESCRIPTION
**Pull request by cjpillsbury**

### CODE REVIEW

File: examples/vanilla-ts-esm/public/mux-uploader-simple.html [hunk 1]
```diff
@@ -53,23 +53,30 @@ <h1><a href="/">Elements</a></h1>
...
+        <mux-uploader pausable></mux-uploader>
     </div>
 
     <a href="../">Browse Elements</a>
 
     <script>
       const input = document.querySelector('input[type="text"]');
       const muxUploader = document.querySelector('mux-uploader');
-      input.addEventListener('change', () => {
+      const updateEndpoint =  () => {
```
Comment (cjpillsbury) [inline examples/vanilla-ts-esm/public/mux-uploader-simple.html:69]: NOTE: changed this to make it easier to dump a `value` in the input while testing locally.

File: packages/mux-uploader/src/layouts/block.ts [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import { document } from '../polyfills';
 
+import '../mux-uploader-drop';
```
Comment (cjpillsbury) [inline packages/mux-uploader/src/layouts/block.ts:3]: NOTE: Safety first.

File: packages/mux-uploader/src/mux-uploader.ts [hunk 3]
```diff
@@ -72,10 +72,11 @@ interface MuxUploaderElement extends HTMLElement {
 
 class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderElement {
   static get observedAttributes() {
-    return ['no-drop', 'no-progress', 'no-status', 'no-retry', 'max-file-size'];
+    return ['pausable', 'no-drop', 'no-progress', 'no-status', 'no-retry', 'max-file-size'];
   }
 
   protected _endpoint: Endpoint;
+  protected _upload?: UpChunk;
```
Comment (cjpillsbury) [inline packages/mux-uploader/src/mux-uploader.ts:79]: NOTE: need to keep this in an instance level prop now for pause/resume functionality.

**Review by luwes — APPROVED** [review]
code looks good! didn't test

**Pull request merged**